### PR TITLE
keyun-fix-team-member-inactive

### PR DESCRIPTION
**Pull request by keyunhuangg**
Screenshot 2023-12-30 at 2 55 17 PM

Video Link: https://www.loom.com/share/58f6a3eabdfe4fa384674f67f58bd297?sid=4f460680-889c-4132-872b-4e487c9ce85e 
## Related PRS (if any):
None
…

## Main changes explained:
- In allTeamsAction.js, isActive parameter was added to the addTeamMember function
- onAddUser updated to pass the isActive attribute of the user to the addTeamMember function

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin/owner user
5. go to otherLink→ Team, and add an activate user to a team
6. verify that the user status is activate with a green circle next to the name without the page being refresh
7. Try to add an inactivate user and it will not show up in search result

## Screenshots or videos of changes:
BEFORE:
https://www.loom.com/share/58f6a3eabdfe4fa384674f67f58bd297?sid=4f460680-889c-4132-872b-4e487c9ce85e

NOW:
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/124097392/d39796b9-8b3f-45ae-9feb-18146027a20c

### CODE REVIEW

**Review by bienzguoa — DISMISSED** [review]
Works as intended! Code changes look clear.
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92126559/dc358c40-aa7e-4ba0-9686-7f4681008377

**Review by Haoj1 — DISMISSED** [review]
@keyunhuangg  Everything looks great! Inactive user would not be in the search box

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/99279909/81ba8fb3-b5b2-409d-ad2a-8b4d080e6e5d

**Review by abdel-lall — DISMISSED** [review]
Works as describe. Nice job!

Screenshot 2023-12-30 182521

**Review by Alforoan — DISMISSED** [review]
Everything worked as intended. Nice!


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118868270/56c182fe-e37c-41be-9bbc-6e3584a9d1ff

File: src/actions/allTeamsAction.js [hunk 1]
```diff
@@ -214,12 +214,12 @@ export const deleteTeamMember = (teamId, userId) => {
...
  * Adding an existing user to team
  */
-export const addTeamMember = (teamId, userId, firstName, lastName, role, addDateTime) => {
+export const addTeamMember = (isActive, teamId, userId, firstName, lastName, role, addDateTime) => {
   const requestData = { users: [{ userId, operation: 'Assign' }] };
   const teamMemberAddPromise = axios.post(ENDPOINTS.TEAM_USERS(teamId), requestData);
   return async (dispatch) => {
     teamMemberAddPromise.then(() => {
-      dispatch(teamMemberAddAction({ _id: userId, firstName, lastName, role, addDateTime }));
+      dispatch(teamMemberAddAction({ isActive, _id: userId, firstName, lastName, role, addDateTime }));
```
Comment (Renanluizssx) [inline src/actions/allTeamsAction.js:222]: it´s a good change

**Review by Renanluizssx — DISMISSED** [review]
It's correct, Nice.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/91035340/109f4273-cf19-4667-af1f-6680ee07e989

**Comment by DragonSlayer2401** [comment]
Your PR works as expected. I was able to assign an active user and see the green circle without refreshing. Please watch the overview video below for more details:


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/102876103/1b9da34e-a60b-4620-897d-2e1c8866af42

**Comment by Haoxiang310** [comment]
I tested your PR with admin and owner role. Everything works as intended. Nice work!
For admin role:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/135764868/73420e4e-3bda-4d28-bb25-8f22ef141aff

For owner role:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/135764868/84f868c5-f907-40c9-ab81-3e36e0acff3f

**Comment by yuhuimin1996** [comment]
Hi @keyunhuangg, I've tested this PR using both owner account and admin account. Both of them worked well as you described. Good job!
owner user:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/108281570/f9be96d2-4aac-4458-9ad4-e3af7c255556
admin user:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/108281570/dcc9453d-90a5-4c75-89f6-36765aae366f

**Comment by Sophie-lei** [comment]
I log in as an admin account, and it works as described

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/70113822/85f16ae2-e60f-4d8f-8734-b89ce08e17f5

**Review by sucheta90 — COMMENTED** [review]
Hello @keyunhuangg, I, in my capacity as an Administrator, have examined this pull request. During the process of adding members to the team, each member was initially marked as an Active member. Nevertheless, it appears that the members were not effectively added to the Team in the Database, as evident in the video below.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/101291283/722744b0-c2d7-4550-b6cb-02d2af79b20a

On further investigating the networks's tab I found that the Team's page is send continuous requests to the backend server. I am not sure if this is normal. Please refer to the video.


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/101291283/be1b0752-7e89-477d-a9c3-283a340a044a

**Pull request closed without merging**